### PR TITLE
replace graphql playground with tyk bundled graphiql

### PR DESCRIPTION
**Pull request by spetrunin**
## Description

Replaced graphql playground handlers provided by graphql-go-tools with a bundled version of GraphiQL with Explorer

## Related Issue

https://tyktech.atlassian.net/browse/TT-2120

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Refactoring or add test (improvements in base code or adds test coverage to functionality)

## Checklist

- [ ] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). If pulling from your own
      fork, don't request your `master`!
- [ ] Make sure you are making a pull request against the **`master` branch** (left side). Also, you should start
      *your branch* off *our latest `master`*.
- [ ] My change requires a change to the documentation.
  - [ ] If you've changed APIs, describe what needs to be updated in the documentation.
  - [ ] If new config option added, ensure that it can be set via ENV variable
- [ ] I have updated the documentation accordingly.
- [ ] Modules and vendor dependencies have been updated; run `go mod tidy && go mod vendor`
- [ ] When updating library version must provide reason/explanation for this update.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] Check your code additions will not fail linting checks:
  - [ ] `go fmt -s`
  - [ ] `go vet`

### CODE REVIEW

**Review by alephnull — CHANGES_REQUESTED** [review]
Since `templates/playground/index.html` is a new static file, it needs to be included in three places:
- deb and rpm files
- slim and std docker images
- tarballs
The good news is that this is all configured in one place, `.goreleaser.yml`. I'm happy to pair with you and make the changes in this PR itself. It shouldn't take more than an hour I think.

**Comment by buger** [comment]
@alephnull I thought that we include the full folder. And since we added files to existing folder (https://github.com/TykTechnologies/tyk/tree/master/templates), it should just work https://github.com/TykTechnologies/tyk/blob/master/.goreleaser.yml#L82

**Comment by buger** [comment]
Ah I see it looks only for json files now.

File: templates/playground/index.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  <script>
+    const apiUrl = "{{.Url}}";
+    const schema = {{.Schema}};
+    const GraphiQlInitInstance = new window.GraphiqlInit({
+      containerId: 'root',
+      url: apiUrl,
+      schema: schema
+    });
+  </script>
+</html>
```
Comment (jay-deshmukh) [inline templates/playground/index.html:31]: LGTM

**Review by jay-deshmukh — COMMENTED** [review]
LGTM

**Review by pvormste — APPROVED** [review]
Code LGTM - just consider @alephnull feedback

**Comment by buger** [comment]
Looks good

**Review by buger — APPROVED** [review]

**Review by alephnull — APPROVED** [review]
LGTM :rocket:

**Comment by alephnull** [comment]
Int image failure is due to gromit being down while I upgrade the integration envs.

**Comment by spetrunin** [comment]
manually cherry-picked into release-3

**Pull request merged**